horace_core/sqw/file_io/@dnd_binfile_common/private/get_fopen_permission_.m

```diff
@@ -0,0 +1,12 @@
+function permission = get_fopen_permission_(permission)
```

Comment:
no tests for this function

Reply:
This is a private function, so I've tested at a higher level in https://github.com/pace-neutrons/Horace/pull/263/commits/f2d3d0806fa3911bb22859111241eac0b70de43b

Reply:
You've tested the get-permission / is_activated loop in those tests - which are good tests - but not that read permission `== rb,` and write permission `== rb+`.

That's probably sufficient for now as testing at the lower level would need a big restructure.

Reply:
I've stuck a check in for `rb` and `rb+`, I didn't put it in originally because it's slightly messy to do, but it's definitely worth having